lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -18,3 +18,14 @@
 .nx-radio__input:checked + .nx-radio__circle .nx-radio__outer-circle {
   fill: none;
 }
+
+.nx-radio-checkbox__input:checked:disabled {
+  + .nx-radio-checkbox__control.nx-radio__circle {
+    stroke: var(--nx-color-interactive-border-disabled);
+  }
+  
```

Comment:
```suggestion

```

Reply:
> Might want to also configure your editor to either show trailing whitespace or to automatically remove it from changed lines.

VS Code has been configured to remove trailing whitespaces.
7b24206ff7820928b158a88caa71208fc898b12e